### PR TITLE
[6.0.0] Added custom login documentation

### PR DESCRIPTION
**Pull request by divyaamunugama**
## Purpose
$subject

### CODE REVIEW

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Customize login portal
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:1]: Have we used sentence case in other IS docs?
Let's use whatever is consistent

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+You can customize the layout and branding of user interfaces like the login page, register page, username and password recovery pages, and single sign-on pages.
+
+All UIs are available in the `authenticationendpoint`, `accountrecoveryendpoint`, and `x509certificateauthenticationendpoint` apps located in `<IS_HOME>/repository/deployment/server/webapps` folder.
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:5]:
```suggestion
All UIs are available in the `authenticationendpoint`, `accountrecoveryendpoint`, and `x509certificateauthenticationendpoint` apps located in the `<IS_HOME>/repository/deployment/server/webapps` folder.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+Navigate to `webapps/authenticationendpoint` and create a new folder named `extensions`.
+
+!!! note
+    Repeat the above step for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+
+### Step 2: Create the basic custom layout resources
+
+To create the basic custom layout resources:
+
+1. On the `authenticationendpoint` folder, navigate to the created `extensions` folder and create the `layouts` folder and the `customs` folders recursively.
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:75]:
```suggestion
1. In the `authenticationendpoint` folder, navigate to the created `extensions` folder and create the `layouts` and `customs` folders recursively.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+    1. Copy the code snippet from the [html file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/body.html) into the `body.html` file.
+
+    2. Copy the code snippet from the [css file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/style.css) into the `styles.css` file.
+
+    3. Copy the code snippet from the [javascript file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/script.js) into the `script.js` file.
+
+    4. Copy the [illustraction.svg](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/illustration.svg) file into the `assets` folder.
+
+2. Navigate to the `webapps/authenticationendpoint/includes` folder and copy the `product-title.jsp` and `product-footer.jsp` files to the `webapps/authenticationendpoint/extensions` folder.
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:107]:
```suggestion
2. Navigate to the `webapps/authenticationendpoint/includes` folder and copy the `product-title.jsp` and `product-footer.jsp` files to the `webapps/authenticationendpoint/extensions` folder as follows:
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+    1. Copy content from the [`jsp` file containing the project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/project-title-1.jsp) to the `webapps/authenticationendpoint/extensions/product-title.jsp` file of the app.
+
+    2. Copy content from the [`jsp` file containing the project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-1.jsp) to the `webapps/authenticationendpoint/extensions/product-footer.jsp` file of the app.
+
+
+    !!! note
+        Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+        
+        The files used for step 2.a and 2.b are as follow:
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:117]:
```suggestion
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+2. Navigate to the `webapps/authenticationendpoint/includes` folder and copy the `product-title.jsp` and `product-footer.jsp` files to the `webapps/authenticationendpoint/extensions` folder.
+
+    1. Copy content from the [`jsp` file containing the project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/project-title-1.jsp) to the `webapps/authenticationendpoint/extensions/product-title.jsp` file of the app.
+
+    2. Copy content from the [`jsp` file containing the project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-1.jsp) to the `webapps/authenticationendpoint/extensions/product-footer.jsp` file of the app.
+
+
+    !!! note
+        Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:115]:
```suggestion
        Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well. You will be copying the following files:
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+        The files used for step 2.a and 2.b are as follow:
+
+        - [Project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-title-2.jsp).
+
+        - [Project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-2.jsp)
+
+3. Refresh the browser and check out the added custom layout.
+
+    !!! tip
+        In certain cases, the server might stop working after adding changes. In such a case, restart the server to check the changes.
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:126]:
```suggestion
        In certain cases, the server might stop working after adding changes. In such cases, restart the server to check the changes.
```

File: en/docs/references/extend/rebranding/customizable-login-portal.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+## Build the app
+
+To build the apps from the `identity-apps` source code:
+
+1. Add the developed custom layout folder into the `<IDENTITY_APPS_SOURCE_CODE>/components/login-portal-layouts/layouts` folder.
+
+2. Build the source code. The custom layout will be moved to the corresponding locations of `authenticationendpoint`, `accountrecoveryendpoint`, and `x509certificateauthenticationendpoint` apps.
```
Comment (nilminiwso2) [inline en/docs/references/extend/rebranding/customizable-login-portal.md:134]:
```suggestion
2. Build the source code. 

The custom layout will be moved to the corresponding locations of `authenticationendpoint`, `accountrecoveryendpoint`, and `x509certificateauthenticationendpoint` apps.
```

File: en/mkdocs.yml [hunk 9]
```diff
@@ -665,7 +665,7 @@ nav:
         - Re-brand WSO2 Identity Server UIs: references/extend/rebranding/rebrand-identity-server-uis.md
         - Re-brand the Default Login Page: references/extend/rebranding/rebrand-login-pages-for-service-providers.md
         - Re-brand the SSO Redirection Page: references/extend/rebranding/rebrand-the-sso-redirection-page.md
-        #- Customize Login page: references/extend/rebranding/customizable-login-portal.md
+        - Customize Login page: references/extend/rebranding/customizable-login-portal.md
```
Comment (nilminiwso2) [inline en/mkdocs.yml:668]:
```suggestion
        - Customize Login Pages: references/extend/rebranding/customizable-login-portal.md
```

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/references/extend/rebranding/customizable-login-portal.md (hunk 1 above)
Comment (divyaamunugama) [reply en/docs/references/extend/rebranding/customizable-login-portal.md:1]: I'll update this we usually use the camel case.

**Pull request merged**